awx/ui_next/src/screens/Host/HostList/HostListItem.test.jsx

```diff
@@ -32,13 +30,12 @@ describe('<HostsListItem />', () => {
         detailUrl="/host/1"
         onSelect={() => {}}
         host={mockHost}
-        onToggleHost={onToggleHost}
       />
     );
   });
 
   afterEach(() => {
-    jest.clearAllMocks();
```

Comment:
why remove `clearAllMocks`?

Reply:
Since we didn't need to mock `onToggleHost`, I didn't see any other mock calls or instances to call `clearAllMocks` on.